### ISSUE TITLE
Still getting chown problems on run

### ISSUE THREAD

**Issue by mark1973ryan** [hashicorp/vault#9082]
Vault Version: latest downloaded on 27/05/20 1.4.2
Operating System/Architecture: Centos8

Vault Config File:
```
{
	"listener": [{
		"tcp": {
			"address": "<dns>:8200",
			"tls_cert_file": "<crt_path>.pem",
			"tls_key_file": "<key_path>.pem"
		}
	}],
	"storage": {
		"file": {
			"path": "/vault/data"
		}
	},
	"ui": true
}
```

Startup Log Output:

Could not chown /vault/config (may not have appropriate permissions)
chown: /vault/config: Permission denied
error loading configuration from /vault/config: open /vault/config: permission denied

Expected Behavior:

On the following command I didn't expect chown issues as previous issues indicate SKIP_CHOWN has a default of true.

docker run -d -p 8200:8200 -v /data/containers/vault:/vault --cap-add=IPC_LOCK vault server

Actual Behavior:

See the log entries above chown was still attempted despite the default value of SKIP_CHOWN

**Comment by mark1973ryan** [comment]
Even ran the container with a user and a group I used to create the files and still got chown errors

**Comment by ncabatoff** [comment]
Hi @mark1973ryan,

I'm not sure why you think SKIP_CHOWN has a default value of true, from what I can tell that's not the case.  Does it work if you use `-e SKIP_CHOWN=1` in your docker command?

**Comment by mark1973ryan** [comment]
Hi,

I read the other issues on the subject that seemed to imply it did. 

The issue is because I cannot have root access I can only get it running using SKIP_CHOWN and SKIP_SETCAP which means I've had to disable mlock. I tried running it with my account but I guess it still tries use the 'vault' user which has no permissions. Due to other security measures and the fact it is a temporary container until we connect to the corp Hashicorp solution isn't a huge issue but not ideal.

Thanks
